### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by Arthusppp**
Update hotkeys translation

### CODE REVIEW

File: files/lang/pl.po [hunk 1]
```diff
@@ -90,12 +90,14 @@ msgid "RESTART"
 msgstr "OD NOWA"
 
 msgid "HEROES"
-msgstr ""
+msgstr "BOHATEROWIE"
```
Comment (ihhub) [inline files/lang/pl.po:93]: Please check how it looks in the game. It is a button in the Kingdom Overview screen. I believe we need to add line separator for this phrase.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Arthusppp , I left one comment in this pull request. Could you please take a look at it?

File: files/lang/pl.po (hunk 1 above)
Comment (Arthusppp) [reply files/lang/pl.po:93]: Screenshot here: 
image

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Arthusppp , thank you so much for this update!

**Pull request merged**